cmd/docs.go

```diff
@@ -36,3 +53,156 @@ func NewCmdDocs() *cobra.Command {
 
 	return cmd
 }
+
+func filePrepender(filename string) string {
+	id := filepath.Base(filename)
+	base := strings.TrimSuffix(id, path.Ext(id))
+	slug := "/cli/commands/" + strings.ToLower(base)
+	title := cases.Title(language.AmericanEnglish).String(strings.ReplaceAll(base, "_", " "))
+
+	return fmt.Sprintf(headerTemplate, id, slug, title, title)
+}
+
+func linkHandler(filename string) string {
+	base := strings.TrimSuffix(filename, path.Ext(filename))
+	return "/cli/commands/" + base
+}
+
+// Currently cobra has a bug where there is an extra line added to the end of the md file when
+// the DisableAutoGenTag = true and this causes pre-commit to have to edit the file after generation
```

Comment:
You could also exclude the generated docs from end-of-file-fixer step in pre-commit.

Reply:
I considered that but the file still has an extra line which may or may not cause issues later, so I up-streamed the fix and when/if it gets merged I will move back to upstream.